bucket/ffdec.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "15.0.0",
+    "description": "Open Source Flash SWF decompiler and editor.",
+    "homepage": "https://github.com/jindrapetrik/jpexs-decompiler",
+    "license": "GPL-3.0-only",
+    "url": "https://github.com/jindrapetrik/jpexs-decompiler/releases/download/version15.0.0/ffdec_15.0.0.zip",
+    "hash": "1f0592b5ee0e079beb19b486787a2b622e3f6a386b11902f832dd3be55c7aff4",
+    "shortcuts": [
```

Comment:
```suggestion
    "bin": "ffdec.jar",
    "shortcuts": [
```

Reply:
Should we add ffdec.jar or ffdec.exe?

Reply:
`ffdec.exe` don't has CLI, it is a pure GUI program.

Reply:
OK, thanks. I assumed it simply called `java -jar ffdec.jar <args>`